includes/admin.form.inc

```diff
@@ -66,5 +66,40 @@ function islandora_social_metatags_admin_form($form, $form_settings) {
     '#description' => t('If the above Image datastreams do not exist for a given object, use a basic default. Use a relative path.'),
     '#default_value' => variable_get('islandora_social_metatags_noimage', '/sites/all/modules/islandora/images/folder.png'),
   );
+  $form['islandora_social_metatags_compound_title'] = array(
+    '#type' => 'select',
+    '#title' => t('Compound Object Title Preference'),
+    '#description' => t('Do you prefer the child title, parent title, or both, concatenated?.'),
```

Comment:
Would reword slightly for clarity. Something to the effect of: "On all child objects, use either the Parent's title, the Child's title, or concatenate both. If choosing Child, the first child object's title will be used on the parent object."